datetime.py

```diff
@@ -0,0 +1,32 @@
+def get_components(datetime:str)->list:                     #get_components checks the length, format and values of date time. Returns the values
```

Comment:
You've done practice on string manipulation here :)

But lets take a different and simpler approach by using Python's `datetime` module. Here's the official docs: https://docs.python.org/3/library/datetime.html but feel free to find simpler and to the point elaborations over the internet.

In this specific case you need `strptime` function from `datetime` module. The acceptable format `dd/MM/yyyy hh:mm:ss` is represented in Python as `%d/%m/%Y %H:%M:%S`.